1_turb_abl_fsi/template_files/nrel5mw_nalu.yaml

```diff
@@ -208,13 +208,11 @@ realms:
       - blade1-HEX
       - blade2-HEX
       - blade3-HEX
-{if(TILT)}
     motion: #tilt done in original mesh location of [0, 0, 0]
       - type: rotation
         angle: 5.0
         axis: [0.0, 1.0, 0.0]
-        centroid: [5.0, 0.0, 0.0] # distance between hub and tower
-{endif}
+        centroid: [5.0191, 0.0, 0.0] # distance between hub and tower
```

Comment:
@psakievich The distance in the horizontal is not 5.0191. I think you are trying to he consistent with the overhang? In which case the centroid would he (5, 0.0, -0.43744). The last value is hub height - tower height - twr2shft.
Or is that value coming from somewhere else?

Reply:
Yes I was trying to make sure we were aligned with the right centroid.  I pulled the overhang value from the openfast files.  The image below makes it look like the value in the file is the radius, but that there might be some additional offset?
<img width="955" alt="image" src="https://github.com/Exawind/exawind-cases/assets/12102506/188e953c-2586-40a9-b544-99ec4e71b669">

Reply:
Overhang is indeed the distance, it is just not the horizontal distance. If we do a `ncdump -v hub_ref_pos <turbine output netcdf file>` we get the position of the hub which tells us that the horizontal distance is `5`. Then there is the vertical offset between `hub height` which is `90m` for the 5-MW, and the `Twr2Shft + TowerHt` about which I believe the tilt happens. 

https://github.com/Exawind/exawind-cases/blob/main/1_turb_abl_fsi/template_files/NRELOffshrBsline5MW_Onshore_ElastoDyn_BDoutputs.dat 

The difference in the hub height and `Twr2Shft + TowerHt` is where I believe the vertical offset comes from. The distance `sqrt(5^2 + 0.43744^2)` then turns out to be `5.0191`.

Reply:
The more I look at this, the more I think we need to tweak our positioning some more. I guess that's a potential topic for today's meeting.  🙂

Reply:
We could possibly be missing something more. However a rotation (tilt), should reduce the horizontal distance, instead of increasing it from the initial distance of 5, no?

Reply:
Would you agree there is a vertical offset, regardless of whether the rotor is pre or post tilt?

Reply:
There is an abuse of terminology here, but checkout table 1-1 and table 4-1 https://www.nrel.gov/docs/fy09osti/38060.pdf

Reply:
That is the opposite of what I am trying to describe.  Here is a diagram of what I am trying to explain.  The 5m is the reduction from 5.0191 m which is the initial distance (pre-tilt)
<img width="955" alt="image" src="https://github.com/Exawind/exawind-cases/assets/12102506/34eb668d-205d-409c-b100-68d393287a86">

Reply:
I see. I believe the line from the hub center to point of rotation is not a horizontal line, even pre-tilt, i.e., I don’t think the rotation is about z = 90, it is about z = 90 - 0.43744

Reply:
I did do an openfast only run with 10 degree tilt, and the `hub_ref_pos` does change, so I think you are right. What I am struggling with is, the center of rotation in `z`. Maybe it is `[5.0191, 0.0, -0.43744]` ?

Reply:
I did a 0 tilt openfast only run, and the `hub_ref_pos` was `1794.9809, 1800, 89.56256`.

Reply:
Yeah I think we need to make a few more adjustments to the position.

Reply:
I think your change is correct. We also need to change - `displacement: [1794.9809, 1800, 89.56256]`